### PR TITLE
Add nested virtualization feature

### PR DESCRIPTION
**Pull request by dsrivastavv**
Add nested virtualization feature for nested testing scenarios

### CODE REVIEW

**Comment by dsrivastavv** [comment]
@squirrelsc @LiliDeng I am currently compiling the list from https://docs.microsoft.com/en-us/azure/virtual-machines/acu. Please let me know if you know of any other source for getting this information

File: lisa/sut_orchestrator/azure/platform_.py [hunk 1]
```diff
@@ -1648,6 +1649,33 @@ def _resource_sku_to_capability(  # noqa: C901
...
+            "standardDDSv4Family",
+            "standardEv3Family",
+            "standardESv3Family",
+            "standardEv4Family",
+            "standardESv4Family",
+            "standardEDv4Family",
+            "standardEDSv4Family",
+            "standardFSv2Family",
+            "standardMSFamily",
+            "standardMSMediumMemoryv2Family",
```
Comment (LiliDeng) [inline lisa/sut_orchestrator/azure/platform_.py:1669]: there are so many vm size support nested-virtualization. e.g.
https://docs.microsoft.com/en-us/azure/virtual-machines/fx-series
https://docs.microsoft.com/en-us/azure/virtual-machines/m-series

Comment (dsrivastavv) [reply lisa/sut_orchestrator/azure/platform_.py:1669]: They all would be covered since they belong to the same family

**Comment by squirrelsc** [comment]
The change looks good to me, please work with Lili on comments.

**Review by squirrelsc — APPROVED** [review]

**Comment by LiliDeng** [comment]
for nested cases, we need add NestedVirtualization in requirement, right? @somil55

**Comment by dsrivastavv** [comment]
> for nested cases, we need add NestedVirtualization in requirement, right? @somil55

Yes Lili! But I haven't enabled them yet. I wanted to confirm the list we have here against the vm names in database where kvm tests pass. Will enable once I confirm this list is exhaustive

**Pull request merged**